### PR TITLE
Location fix

### PR DESCRIPTION
**Pull request by marcvaneijk**
Changed resource location from variable to resourceGroup().location.
Deleted location variable from deployment template.

### CODE REVIEW

File: bosh-setup/azuredeploy.json [hunk 1]
```diff
@@ -233,7 +232,7 @@
       "apiVersion": "[variables('apiVersion')]",
       "type": "Microsoft.Network/publicIPAddresses",
       "name": "[concat(parameters('vmName'),'-devbox')]",
-      "location": "[variables('location')]",
+      "location": "resourceGroup().location]",
```
Comment (singhkay) [inline bosh-setup/azuredeploy.json]: Missing '[' here

**Comment by marcvaneijk** [comment]
Added missing bracket

**Comment by bingosummer** [comment]
@marcvaneijk 

Could you please explain the reason the location should be fixed? Thanks.

**Comment by marcvaneijk** [comment]
@bingosummer we're standardizing the templates with a base set from the recently updated guidelines https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

See comment #1727

**Comment by bingosummer** [comment]
@marcvaneijk 

However, in `bosh-setup`, we set a variable to `resourceGroup().location` and use the variable to specify the location, which should follow the best practices.

It makes more sense for me because I don't need to use `resourceGroup().location` in each resource.

**Comment by marcvaneijk** [comment]
@bingosummer yes, that is fine too. I'm just standardizing. You can deviate of course. As long as fixed values or params containing fixed values for the location are not used, unless there is a good reason for it (e.g. resource only available in one region). VS also sets the `resourceGroup().location` on resources added through the wizard.

**Comment by bingosummer** [comment]
Got it. Thanks @marcvaneijk

**Pull request merged**